client/http/client.go

```diff
@@ -458,6 +466,44 @@ func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRan
 	return &regionStats, nil
 }
 
+// SetStoreLabels sets the labels of a store.
+func (c *client) SetStoreLabels(ctx context.Context, storeID int64, storeLabels map[string]string) error {
+	jsonBody, err := json.Marshal(storeLabels)
```

Comment:
Better align with the naming style of other functions.

```suggestion
	jsonInput, err := json.Marshal(storeLabels)
```